src/main/java/fr/insalyon/creatis/gasw/plugin/executor/dirac/execution/DiracMonitor.java

```diff
@@ -148,6 +172,8 @@ public void run() {
                                     case Stalled:
                                         job.setStatus(GaswStatus.STALLED);
                                         break;
+                                    case Completed:
+                                        logger.info("Dirac Monitor: job \"" + job.getId() + "\" has status \"" + status + "\"");
```

Comment:
Shouldn't case "Completed" be handled the same as "Done" ? I would say that these two Dirac statuses are equivalent from VIP's point of view

Reply:
I think that failed jobs can also go through the "COMPLETED" status.
Also I think we should wait for the "DONE" status for the output and error files to be available.